docs/modules/transform-sync/how-it-works.md

```diff
@@ -0,0 +1,63 @@
+# How the Transform Synchronization Feature Module works
+
+## What is transform
+
+The transform of an entity describes its location and rotation. Worker-instances generally use it to run physics simulations or render entities for a client. Note that the SpatialOS representation of transform is _not_ the same as Unity's representation of transform.
```

Comment:
Orientation is more accurate.
Also possibly velocity.

Reply:
The first sentence refers to transform as a concept, I guess the idea was to present it in a way that's familiar to Unity users - i.e. - the Unity transform component has position and rotation immediately available - then move into how we treat transform 

How would you structure this section?

Reply:
I doesn't matter that much. If we think that is easier to understand for unity users then it's better.